app/models.py

```diff
@@ -14,17 +14,26 @@ class Recruiter(models.Model):
     first_name = models.CharField(max_length=50)
     last_name = models.CharField(max_length=50)
     email = models.CharField(max_length=50)
-    organization = models.ForeignKey(Organization, on_delete=models.CASCADE)
+    organization = models.CharField(max_length=50)
     is_superadmin = models.BooleanField(default=False)
 
+@receiver(post_save, sender=User)
+def create_user_profile(sender, instance, created, **kwargs):
```

Comment:
I to nie powinno dziać się w sygnałach https://lincolnloop.com/blog/django-anti-patterns-signals/